README.md

```diff
@@ -1 +1,20 @@
-# syncer
\ No newline at end of file
+# syncer
+
+Syncs galaxy integration forks with upstream
+
+## Example Usage (workflow)
+```yaml
+name: sync-upstream
+on:
+  schedule:
+    - cron: '30 5,17 * * *'
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: GOG-Nebula/syncer@v0
```

Comment:
I dont see v0 specified in action.yml. Could you add?

Reply:
I don't think the version is specified in `action.yml`. It is defined as a tag on a commit. But I realise just using the branch name is better for now.